_posts/2019-09-15-guide_create_surf_ramps.md

```diff
@@ -319,12 +328,32 @@ The vmf created during this guide is available [here](/assets/mapsrc/surf_ramp_t
   
     
 ## Modeling Ramps
+<img src="/assets/images/ramps_guide/headers/nebula.jpg" alt="Model Header" style="display: block; margin: auto">
+
+While a lot can be accomplished with brushes, complex brushes and ramps will sometimes reach the limits of what can be done with them. 
+At this point it may be time to look into making the ramps into models.  
+  
+Normally this doesn't need to be done until late in development when doing an art pass for details, 
+but it tends to be a lot more involved and requires experience with modeling software.  
+  
+Model collision can also be prone to having far more rampbugs. 
```

Comment:
Model collision without clipping right?

Reply:
Added clarification in commit [
24b47d2](https://github.com/momentum-mod/docs/pull/20/commits/24b47d20defd12aca1bfbf29cadd92753ae1d9a9)